.github/workflows/checks.yml

```diff
@@ -81,3 +81,17 @@ jobs:
     - name: Find unreferenced images
       run: |
         ./scripts/unreferenced_images.bash
+
+  redirects_check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-python@v5
+      with:
+        python-version: '3.10'
+    - name: Check redirects
+      run: |
+        git diff --name-status $(git describe --tags $(git rev-list --tags --max-count=1)) > git-output.txt
+        python scripts/check-redirects/main.py
```

Comment:
2 questions:
- This file is generating multiple files .txt (...) . Do we need these files?
- Do we have any guidelie, how to fix PR which throws exit 1 from main.py ? What to do next?

Reply:
- text files are used only internally during deployment (redirects for nginx)
- I think writers should run the script manually, check the output and try to fix REDIRECTS file or contact ops if in doubt - whole point of this check is just to prevent merging PR with invalid redirects

Reply:
Is it possible to fix that redirects on nginx before merging PR ? And then rerun check and everything will be ok -> merge PR?  - if yes -> approved for me :)

Reply:
not sure what you mean before merging - these pre-checks are to confirm that REDIRECTS file is valid and complete and hence we can continue with deployment